### PR TITLE
modifications to the content

### PR DESCRIPTION
**Pull request by ptrcarta**
fixed a couple punctuation inaccuracies and rewrote some of the content
in more direct words

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -8,40 +8,36 @@ Finding an apartment in Lausanne with a rent that fits a student budget is a
...
-below and you will understand how those were obtained. Even if Lausanne is not
+In particular you will be enlightened by the following real estate insights
+and you will understand how these were obtained. Even if Lausanne is not
 New York we needed to think of it in smaller units. Therefore, our results
 divide the city into its neighbourhoods – in french
 [_quartiers_][quartiers_lausanne].
 
-- We evaluated the mean rents per _quartier_  and can therefore say that the
-  _quartiers_  _Montriond_ and _Ouchy_  are clearly more expensive
+- the _quartiers_  _Montriond_ and _Ouchy_  are clearly more expensive
```
Comment (yannbolliger) [inline index.md]:
```suggestion
- The _quartiers_  _Montriond_ and _Ouchy_  are clearly more expensive
```
all others start with capital letters

File: index.md [hunk 2]
```diff
@@ -8,40 +8,36 @@ Finding an apartment in Lausanne with a rent that fits a student budget is a
...
-- For each _quartier_, we analysed its ownership pattern. This means we know
-  what type of owner possesses the most properties and also how diverse the
-  owners of _quartier_ are. For example the centre is mainly owned by
-  corporations.
+- The city center is mostly owned by corporations,
+  _Chailly_ is mostly owned by privates.
 
-- Even if some parts of the city are owned by different types of owners, there
-  is no direct relation between owner type and prices. The market seems to
+- There is no direct relation between owner type and prices. The market seems to
```
Comment (yannbolliger) [inline index.md:36]:
```suggestion
- There is no direct relation between the owner type and prices. The market seems to
```

File: index.md [hunk 3]
```diff
@@ -8,40 +8,36 @@ Finding an apartment in Lausanne with a rent that fits a student budget is a
...
   adjust prices uniformly over the ownership types.
 
-- However, the height of a rent is strongly influenced by the distance to the
-  lake Geneva and by the inverse surface of the flat. Smaller flats cost more
+- The price per square meter of rent is strongly influenced by the distance to the
```
Comment (yannbolliger) [inline index.md]:
```suggestion
- The price per square meter of a flat is strongly influenced by the distance to the
```

File: index.md [hunk 4]
```diff
@@ -83,12 +79,12 @@ a small amount of parcels we will group them into 7 types:
  - cooperatives: _registered cooperative companies like Migros, la Mobilière etc._
 
  - foundations and associations: _for example the olympic foundation for
-    cultural heritage._
+cultural heritage._
 
- - PPE: _a swiss thing: single flats owned by private people – in french_
-   proriété par étage.
+ - PPE: _the single flats in a building are owned by different people – in french
+   proriété par étage_.
```
Comment (yannbolliger) [inline index.md]:
```suggestion
_   proriété par étage.
```
French words are italic, but in italic text, the new italic is again normal text...

File: index.md [hunk 5]
```diff
@@ -83,12 +79,12 @@ a small amount of parcels we will group them into 7 types:
...
 
- - individual privates: _just normal people like you and me._
+ - individual privates: _The whole building is owned by a private citizen._
```
Comment (yannbolliger) [inline index.md]:
```suggestion
 - individual privates: _the whole building is owned by a private citizen._
```

Here all others start with downcase.

File: index.md [hunk 6]
```diff
@@ -165,26 +161,25 @@ whether the ownership type also influences the price of accommodation. To give a
...
 
 But if the owner doesn't influence the price what does? Why are prices
-different. You can get an intuition for this by looking at the smoothed rent
+different? You can get an intuition for this by looking at the smoothed rent
 map. Clearly, the parts near the lake are more red. Our next linear regression
 therefore analyses the dependence of the price on the distance to the port of
 Lausanne, [_Ouchy_]. As you can observe in the plot to the left, there is a
 visible correlation. Our model can confirm with a high significance that the
-more you go up hill the cheaper flats get.
+more you go toward the country-side, the cheaper flats get.
```
Comment (yannbolliger) [inline index.md]:
```suggestion
more you go towards the country-side, the cheaper flats get.
```

**Review by yannbolliger — CHANGES_REQUESTED** [review]

File: index.md (hunk 2 above)
Comment (ptrcarta) [reply index.md:36]: je pense que ma version est aussi correcte

File: index.md (hunk 4 above)
Comment (ptrcarta) [reply index.md]: ok but weird

File: index.md (hunk 2 above)
Comment (yannbolliger) [reply index.md:36]: Sure, c'était juste le flow. Mais ça marche aussi.

**Review by yannbolliger — APPROVED** [review]

**Pull request merged**